app/unit-tests/src/org/commcare/android/tests/processing/FormStorageTest.java

```diff
@@ -394,6 +394,7 @@ public class FormStorageTest {
             , "org.commcare.android.database.global.models.GlobalErrorRecord"
 
             ,"org.commcare.android.database.connect.models.ConnectUserRecordV14"
+            ,"org.commcare.android.database.connect.models.PersonalIdCredential"
```

Comment:
can we add a comment above this line saying `//Added in 2.58` similar to other version comments above